rules/windows/defense_evasion_execution_msbuild_started_renamed.toml

```diff
@@ -23,7 +23,7 @@ type = "query"
 
 query = '''
 event.category:process and event.type:(start or process_started) and
-  (pe.original_file_name:MSBuild.exe or winlog.event_data.OriginalFileName:MSBuild.exe) and
+  (process.pe.original_file_name:MSBuild.exe or winlog.event_data.OriginalFileName:MSBuild.exe) and
```

Comment:
This was the only actual logic change to any rules - necessary due to field name changes in schema

Reply:
Beats _may_ have also been updated as well, in which case, we'd be able to remove the winlog field altogether - will check on that

Reply:
as of 7.8.0, this is still not the case. Can re-check closer to 7.9.0 release